toolkit/tools/internal/timestamp_v2/timestamp_v2_safe_logger.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Should we merge this into the main logger package? Just chucked it here for early dev work.